crates/starfish/core/src/block_header.rs

```diff
@@ -272,7 +269,6 @@ impl BlockRef {
     }
 }
 
-// TODO: re-evaluate formats for production debugging.
```

Comment:
#8152 issue created to discuss this TODO as well as the following few Display and Debug related TODOs